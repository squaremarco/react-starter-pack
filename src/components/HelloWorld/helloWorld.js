import React from 'react';
import styled from 'styled-components';

const StyledHeading = styled.h1`
  text-align: center;
  color: #4c4c4c;
`;

export default () => <StyledHeading>Hello, World!</StyledHeading>;
