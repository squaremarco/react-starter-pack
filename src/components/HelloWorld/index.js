import React from 'react';
import Heading from 'Components/StyledComponents/Heading';

const HelloWorld = () => <Heading>Hello, World!</Heading>;

export default HelloWorld;
