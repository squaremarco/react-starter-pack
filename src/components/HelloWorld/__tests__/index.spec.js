import React from 'react';
import { render, cleanup } from '@testing-library/react';
import HelloWorld from '..';

afterEach(cleanup);

describe('HelloWorld', () => {
  it('Snapshot test', () => {
    const { asFragment } = render(<HelloWorld />);
    expect(asFragment()).toMatchSnapshot();
  });
});
