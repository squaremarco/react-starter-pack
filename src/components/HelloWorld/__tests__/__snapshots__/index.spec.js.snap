// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld Snapshot test 1`] = `
<DocumentFragment>
  .c0 {
  text-align: center;
  color: #4c4c4c;
}

<h1
    class="c0"
  >
    Hello, World!
  </h1>
</DocumentFragment>
`;
